components/cam/bld/config_files/definition.xml

```diff
@@ -315,5 +315,47 @@ that setting to allow for cross-compilation, and for instances where the
 $OSNAME value is too generic.  For example, currently on both cray-xt and
 bluegene systems $OSNAME has the value "linux".
 </entry>
+<entry id="crm" valid_values="0,1" value="0">
```

Comment:
`crm` and `use_MMF` flags appear to be duplicates...do we need both?

Reply:
I think I meant to delete this. I'm pretty sure I removed the crm variable from the configure code.

Reply:
Looks like I just forgot to delete that after I deleted the spots where it was used, so that's an easy fix.